src/index.html

```diff
@@ -1,17 +1,197 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
+    </meta>
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="container">
+        <legend>Personal information</legend>
+
+        <label class="box">
+          Surname:
+
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="box">
+          Name:
+
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="box">
+          How old are You?
+
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="box">
+          Full date of birth:
+
+          <input
+          type= "date"
+          id ="date"
+          name ="date"
+          >
```

Comment:
```suggestion
          <input
            type= "date"
            id ="date"
            name ="date"
          >
```

[CODE STYLE] - Remember about correct indentation between parent and child elements. Each level of nesting, including text, contained inside the element, requires 2-space offset.